client/lib/i18n-utils/test/localize-path.js

```diff
@@ -0,0 +1,103 @@
+import { localizePath } from '../localize-path';
+
+describe( '#localizePath', () => {
+	test( 'should not change path for `en`', () => {
+		[ '/', '/de/', '/start/', '/wp-login.php?action=lostpassword' ].forEach( ( path ) => {
+			expect( localizePath( path, 'en' ) ).toEqual( path );
+		} );
+	} );
+
+	test( 'should change path if specified', () => {
+		expect( localizePath( '/wp-admin', 'en', false ) ).toEqual( '/wp-admin' );
+		expect( localizePath( '/wp-admin', 'de', true ) ).toEqual( '/wp-admin' );
+		expect( localizePath( '/wp-admin', 'de', false ) ).toEqual( '/wp-admin' );
+		expect( localizePath( '/wp-login.php', 'es', false ) ).toEqual( '/wp-login.php' );
+
+		expect( localizePath( '/log-in', 'es' ) ).toEqual( '/log-in/' );
+		expect( localizePath( '/log-in', 'es', false ) ).toEqual( '/log-in/es/' );
+
+		expect( localizePath( '/new', 'es' ) ).toEqual( '/new/' );
+		expect( localizePath( '/new', 'es', false ) ).toEqual( '/new/es/' );
+
+		expect( localizePath( '/setup', 'es' ) ).toEqual( '/setup/' );
+		expect( localizePath( '/setup', 'es', false ) ).toEqual( '/setup/es/' );
+
+		expect( localizePath( '/start', 'es' ) ).toEqual( '/start/' );
+		expect( localizePath( '/start', 'es', false ) ).toEqual( '/start/es/' );
+
+		expect( localizePath( '/start/user', 'es' ) ).toEqual( '/start/user/' );
+		expect( localizePath( '/start/user', 'es', false ) ).toEqual( '/start/user/es/' );
```

Comment:
```suggestion
		expect( localizePath( '/start/user', 'es', false ) ).toEqual( '/start/user/es' );
		expect( localizePath( '/start/user/', 'es', false ) ).toEqual( '/start/user/es/' );
```
`localizePath` shouldn't be adding trailing slashes (we redirect to strip them), but lets not remove them if provided either.

Reply:
Sounds good! 
I pushed a fix for this and updated the rest of the tests.